pkg/systemstatsmonitor/host_collector.go

```diff
@@ -26,13 +26,12 @@ import (
 )
 
 type hostCollector struct {
-	tags       map[string]string
-	uptime     *metrics.Int64Metric
-	lastUptime int64
+	tags   map[string]string
```

Comment:
Are the metrics changes related to stackdriver? If not, do you mind making them a separate commit or PR, so that it is easier to review?

Same question applies to other metrics related file changes.

Reply:
Yes, this refactoring is related to the Stackdriver PR. Here is the incentive:

1. The OpenCensus Stackdriver exporter determines the metric endpoint [1] from view name [2] by calling the [`GetMetricType()` option](https://godoc.org/contrib.go.opencensus.io/exporter/stackdriver#Options).

1. So we must keep a mapping from view name to metric endpoint name.

1. Before this refactoring, each individual problem daemon defines their own view name at run time (specified by the config files specified for them). It's hard to keep a global record for the above mapping.

This refactoring is simply to track that mapping.

I separated it into a separate commit.

[1] "compute.googleapis.com/guest/xxx", i.e. the name recognized by Stackdriver Monitoring service.
[2] "disk/io_time", i.e. the `viewName` field we used when calling `NewFloat64Metric()`